<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>430</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>661</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="selectionRectVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionStop"/>
   </widget>
   <addaction name="menuConnect"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect">
   <property name="text">
    <string>connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>stop</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
